### PR TITLE
Feature/pelagos 4803 update dataland coldstorage

### PR DESCRIPTION
**Pull request by fightingtexasaggie**
To test, merge in 4802 then use dataset-review to set size/count and view in dataland. (If for some reason you want to use the cold-storage command instead to set the count/size, make sure and re-accept in dataset-review.)

### CODE REVIEW

File: templates/Dataland/v2/download-card.html.twig [hunk 1]
```diff
@@ -28,14 +54,16 @@
...
                         {{ dataset.datasetSubmission.distributionFormatName }}
                     </p>
-                    <hr>
-                    <p class="card-text text-dark">
-                        <a href="#individual-files">Individual Files
-                            <img src="{{ asset('build/images/down-arrow.svg') }}"><br />
-                        </a>
-                    </p>
-                {% endif %}
+                    {% if dataset.availabilityStatus == constant('App\\Entity\\DatasetSubmission::AVAILABILITY_STATUS_PUBLICLY_AVAILABLE') %}
```
Comment (praneethpr) [inline templates/Dataland/v2/download-card.html.twig:57]: I don't think this check is necessary as it is the same as line 1

Comment (fightingtexasaggie) [reply templates/Dataland/v2/download-card.html.twig:57]: It's there in the event the dataset is remotely hosted. We don't want to show the file manager link in that case.

Comment (mickel1138) [reply templates/Dataland/v2/download-card.html.twig:57]: I agree, this line is not needed.

**Review by mickel1138 — APPROVED** [review]
Looks ok.

**Pull request merged**